airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -110,7 +112,10 @@ def _write_local_data_files(self, cursor):
         tmp_file_handles = { self.filename.format(file_no): tmp_file_handle }
 
         for row in cursor:
+            # Convert datetime objects to utc seconds
+            row = map(lambda v: time.mktime(v.timetuple()) if type(v) in (datetime, date) else v, row)
```

Comment:
I learned yesterday that functools.partial is considered more pythonic than lambda.
(http://stackoverflow.com/a/3252425)
not really a big deal, but in case you care.